components/scream/src/mct_coupling/scream_cpl_indices.F90

```diff
@@ -69,48 +69,57 @@ subroutine scream_set_cpl_indices (x2a, a2x)
     do i=1,num_scream_imports
       import_vector_components(i) = -1
       import_constant_multiple(i) = 1
-      do_import_during_init(i) = .true.
+      do_import_during_init(i) = .false.
```

Comment:
This is an unrelated change, but I snuck it in here. Makes more sense to me to default to `false` and anyone adding an import/export field should have to explicitly tell the coupler they want that field imported/exported during initialization.